lib/stomp_client.ex

```diff
@@ -394,8 +394,8 @@ defmodule StompClient do
           end
         end
 
-      x ->
-        x
+      :partial ->
+        {:partial, message}
     end
```

Comment:
The above change will fail, when return value is {:error, remain}. x -> x was passing both :partial and {:error, remain} as it is, and they were being processed further. One more condition can be added here, but then overall it will be a zero sum game.